### PR TITLE
Added identifier on newSession()

### PR DESCRIPTION
**Pull request by emilm**
I hope this is OK, and that you will accept it. Please tell me if I did the tests wrong or something else!
Perhaps add it on close as well, don't see the point as you should have the info when you have the session ID. :)

### CODE REVIEW

**Comment by TVolden** [comment]
It looks good. 

I don't think it's necessary to store the id in the session since it's not used. 
It's necessary for the SOAP implementation to store the id in the communicator in order to generate the headers correctly.

**Comment by emilm** [comment]
Now the ID is outside.

File: ocpp-common/src/main/java/eu/chargetime/ocpp/Client.java [hunk 1]
```diff
@@ -45,15 +45,17 @@ of this software and associated documentation files (the "Software"), to deal
 public abstract class Client extends FeatureHandler
 {
     private Session session;
+    private final String chargeBoxIdentity;
```
Comment (TVolden) [inline ocpp-common/src/main/java/eu/chargetime/ocpp/Client.java]: Will this variable ever be used? Or should it have a getter and setter?

File: ocpp-common/src/test/java/eu/chargetime/ocpp/test/ServerTest.java [hunk 2]
```diff
@@ -98,10 +99,10 @@ public void newSession_serverIsListening_callbackWithIndex0() {
         server.open(LOCALHOST, PORT, serverEvents);
 
         // When
-        listenerEvents.newSession(session);
+        listenerEvents.newSession(session, IDENTIFIER);
 
         // Then
-        verify(serverEvents, times(1)).newSession(any(UUID.class));
+        verify(serverEvents, times(1)).newSession(any(UUID.class), anyString());
```
Comment (TVolden) [inline ocpp-common/src/test/java/eu/chargetime/ocpp/test/ServerTest.java]: Could match the second parameter with IDENTIFIER.

File: ocpp-common/src/main/java/eu/chargetime/ocpp/Client.java (hunk 1 above)
Comment (emilm) [inline ocpp-common/src/main/java/eu/chargetime/ocpp/Client.java]: No. :)

File: ocpp-common/src/test/java/eu/chargetime/ocpp/test/ServerTest.java (hunk 2 above)
Comment (emilm) [inline ocpp-common/src/test/java/eu/chargetime/ocpp/test/ServerTest.java]: Will do, I will make a new commit and squash it

**Comment by emilm** [comment]
Updated the code

**Comment by TVolden** [comment]
Brilliant, thanks.

**Pull request merged**